src/lexer/tokens.rs

```diff
@@ -19,6 +19,10 @@ pub enum Token {
     #[token("{ref}")]
     PropertyByRef,
 
+    #[token("@CONSTANT")]
```

Comment:
I think we can start phasing out the @... markers since we can use pragmas at this position

Reply:
but no need for a change now, it's just a thought